packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -63,7 +74,38 @@ class Chat extends Widget<Properties> {
   _renderMessageBox(): void {
     const $messageBox = $('<div>').appendTo(this.element());
 
-    this._messageBox = this._createComponent($messageBox, MessageBox, {});
+    const configuration: MessageBoxProperties = {
+      onMessageSend: (text) => {
```

Comment:
The incoming argument is object that contains the following fields: component, event, text, element. Let's rename this argument.

Reply:
[cd52ce11e169517258f5b51a5b14ac75315205da]